lib/include/openamp/rpmsg.h

```diff
@@ -101,7 +101,7 @@ struct rpmsg_device_ops {
 	int (*send_offchannel_nocopy)(struct rpmsg_device *rdev,
 				      uint32_t src, uint32_t dst,
 				       const void *data, int len);
-	int (*release_tx_buffer)(struct rpmsg_device *rdev, void *rxbuf);
+	int (*release_tx_buffer)(struct rpmsg_device *rdev, void *txbuf);
```

Comment:
ok, this is fixed in a different commit then. Can you squash this change into the first commit?